### PR TITLE
Remove ZEMA BNN from repo and update docs accordingly

### PR DESCRIPTION
**Pull request by BjoernLudwigPTB**
We finally removed the package _ZEMA_BNN_ from the repo and since we used it to show how to include notebooks into the docs, we replaced those and finished the introduction of the tutorials. At the same time we made the _README.md_ available as a _Getting started_ document.

### CODE REVIEW

**Comment by BjoernLudwigPTB** [comment]
The docs you can find [here](https://agentmet4fof.readthedocs.io/en/remove_zema_bbn_from_repo/) now

File: agentMET4FOF/ml_uncertainty/bnn.py [hunk 1]
```diff
@@ -1,54 +1,11 @@
...
-
-from sklearn.metrics import f1_score
-from sklearn.metrics import confusion_matrix
-
-from scipy.stats import skew
-from scipy.stats import kurtosis
-from scipy.fftpack import fft
-
-import pandas as pd
+from agentMET4FOF.ml_uncertainty.bnn import *
```
Comment (BjoernLudwigPTB) [inline agentMET4FOF/ml_uncertainty/bnn.py]: This is the actual origin of the strange behaviour of the import optimization . It is clearly a bug in PyCharm, but this statement on its own is kind of irritating. This basically means "import everything from in here" but why should we do that? PyCharm's import optimization resolves this and notices, that there is acutally everything present, if this import is performed and then deletes all other imports already present, but this of course recursively deletes the actual imports.

File: agentMET4FOF/ml_uncertainty/bnn.py [hunk 2]
```diff
@@ -1,54 +1,11 @@
...
 
+from scipy import stats
+from scipy import stats
```
Comment (BjoernLudwigPTB) [inline agentMET4FOF/ml_uncertainty/bnn.py]: This just is a nasty bug-side-effect.

Comment (bangxiangyong) [reply agentMET4FOF/ml_uncertainty/bnn.py]: i just tried to pycharm optimiser (ctrl+alt +o) these days, i find that occassionally it is not that reliable though..

Comment (BjoernLudwigPTB) [reply agentMET4FOF/ml_uncertainty/bnn.py]: Oh really? Could it be, that these cases occurred were as well in conjunction with * imports? I use this feature almost every day without any problem up until this one exceptional case. If there are problems with it or other cases where it is not advisable I would be glad if you could show or tell me.

**Pull request merged**